src/Ledger/Gov.lagda

```diff
@@ -89,7 +89,7 @@ data _⊢_⇀⦇_,GOV'⦈_ : GovEnv × ℕ → GovState → GovVote ⊎ GovPropo
     actionWellFormed a ≡ true
```

Comment:
Since this is a PDF cleanup, line 78 above  (the type signature of the relation) overflows to the right of the page.